app/controllers/nodes_controller.rb

```diff
@@ -1,18 +1,17 @@
 # This controller exposes the REST operations required to manage the Node
 # resource.
 class NodesController < NestedNodeResourceController
+  include DynamicFieldNamesCacher
   include NodesSidebar
 
   skip_before_action :find_or_initialize_node, only: [ :sort, :create_multiple ]
   before_action :initialize_nodes_sidebar, except: [ :sort, :create_multiple ]
+  before_action :set_evidence_default_columns, only: :show
```

Comment:
We use `:set_columns` in other controllers. Arguably you could end up here with `set_evidence_columns` and `set_note_columns` at some point, but I don't think we need the `_default_` part